### PR TITLE
Some fixes for methanol

### LINKED ISSUE #237: Methanol Confusion
The unravelling of the methanol buses and links don't make any sense https://github.com/PyPSA/pypsa-ariadne/blob/c5e5b943b53e1b53f9678f8827446c92c84cde0d/workflow/scripts/modify_prenetwork.py#L489
`DE industry methanol` should be a bus with corresponding load.
There is also a link from `DE industry methanol` to the same bus to track emissions but only for Germany?

### PR DESCRIPTION
**Pull request by lindnemi**
- Report MeOH as Liquids|Hydrogen (Supersedes #152)
- Correct typo in modify_prenetwork (Closes #237 )

Before asking for a review for this PR make sure to complete the following checklist:

- [ ] Workflow with target rule `ariadne_all` completes without errors
- [ ] The logic of `export_ariadne_variables` has been adapted to the changes
- [ ] One or several figures that validate the changes in the PR have been posted as a comment
- [ ] A brief description of the changes has been added to `Changelog.md`
- [ ] The latest `main` has been merged into the PR
- [ ] The config has a new prefix of the format `YYYYMMDDdescriptive_title`

### CODE REVIEW

File: workflow/scripts/export_ariadne_variables.py [hunk 1]
```diff
@@ -1684,7 +1689,6 @@ def get_secondary_energy(n, region, _industry_demand):
         + var["Secondary Energy|Gases"]
         + var["Secondary Energy|Liquids"]
         + var["Secondary Energy|Solids"]
-        + var["Secondary Energy|Methanol"]
```
Comment (toniseibold) [inline workflow/scripts/export_ariadne_variables.py:1687]: So "Secondary Energy|Methanol" is not reported anymore?

File: workflow/scripts/export_ariadne_variables.py [hunk 2]
```diff
@@ -3898,17 +3902,69 @@ def get_export_import_links(n, region, carriers):
...
+        .sum()
+    )
+
+    assert region == "DE"  # only DE is implemented at the moment
+
+    DE_renewable_oil = renewable_oil_supply.get("DE renewable oil", pd.Series(0))
+    EU_renewable_oil = renewable_oil_supply.get("EU renewable oil", pd.Series(0))
+
+    if DE_renewable_oil.sum() == 0:
+        DE_bio_fraction = 0
```
Comment (toniseibold) [inline workflow/scripts/export_ariadne_variables.py:3918]: Rather renewable fraction? Or is it only from biomass sources?

**Review by toniseibold — CHANGES_REQUESTED** [review]
Minimalistic comments to your PR which makes it difficult for me to review changes especially in the exporter.

File: workflow/scripts/export_ariadne_variables.py (hunk 2 above)
Comment (lindnemi) [reply workflow/scripts/export_ariadne_variables.py:3918]: only bio sources

File: workflow/scripts/modify_prenetwork.py
[the remark this answers is not in the document]
Comment (lindnemi) [reply workflow/scripts/modify_prenetwork.py:1]: good catch

**Comment by lindnemi** [comment]
merged via GitLens (bad idea, messed up history)

**Pull request closed without merging**